source/math/IntSize2D.ooc

```diff
@@ -36,7 +36,7 @@ IntSize2D: cover {
 	minimum: func ~Int (ceiling: Int) -> This { this minimum(This new(ceiling)) }
 	maximum: func ~Int (floor: Int) -> This { this maximum(This new(floor)) }
 	clamp: func (floor, ceiling: This) -> This { This new(this width clamp(floor width, ceiling width), this height clamp(floor height, ceiling height)) }
-	fillEven: static func (other: This) -> This { This new(other width + (other width % 2 == 1 ? 1 : 0), other height + (other height % 2 == 1 ? 1 : 0)) }
+	fillEven: static func (other: This) -> This { This new(other width + (Int odd(other width) ? 1 : 0), other height + (Int odd(other height) ? 1 : 0)) }
```

Comment:
Wait, maybe it's because it didn't work well for negative values... but in that case, I think `Int modulo(other width, 2)` would still read more clearly than this.